docs/pages/reference/cli.mdx

```diff
@@ -276,6 +276,78 @@ Run `tsh help <subcommand>` or see the [Global Flags section](#tsh-global-flags)
 $ tsh --user=joe --login=ec2-user join <session-id>
 ```
 
+### tsh recordings ls
+
+List recorded sessions.
+
+```code
+$ tsh recordings ls [<flags>]
+```
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--from-utc` | 24 hours ago | date | Start of time range in which recordings are listed. Format 2006-01-02. Defaults to 24 hours ago. |
+| `--to-utc` | current | date | Start of time range in which recordings are listed. Format 2006-01-02. Defaults to 24 hours ago. |
+| `--limit` | 50 | number | Maximum number of recordings to show. |
+| `--last` | none | duration | Duration into the past from which session recordings should be listed. Format 5h30m40s |
+
+#### Global flags
+
+These flags are available for all commands `--login, --proxy, --user, --ttl, --identity, --cert-format, --insecure, --auth, --skip-version-check, --debug, --jumphost, --format`.
+Run `tsh help <subcommand>` or see the [Global Flags section](#tsh-global-flags).
+
+#### Examples
+
+```code
+# get the recorded sessions from the last 24 hours
+$ tsh --proxy proxy.example.com recordings ls
+ID                                   Type Participants Hostname Timestamp
+------------------------------------ ---- ------------ -------- -------------------
+b0a04442-70dc-4be8-9308-7b7901d2d600 ssh  jeff         dev       Nov 26 16:36:16 UTC
+c0a02222-70dc-4be8-9308-7b7901d2d600 kube alice                  Nov 26 20:36:16 UTC
+d0a04442-70dc-4be8-9308-7b7901d2d600 ssh  navin        test      Nov 26 16:36:16 UTC
+
+#After retrieving the recorded session can be played with tsh play
+$ tsh play c0a02222-70dc-4be8-9308-7b7901d2d60
+
+# Retrieve recorded sessions that occurred between Nov 1, 2022 to Nov 3, 2022
```

Comment:
```suggestion
# List recorded sessions that occurred between Nov 1, 2022 to Nov 3, 2022
```

"retrieve" sounds like "download" to me. This is just a listing, it's not really "retrieving" anything